### ISSUE TITLE
Fully detectable

### ISSUE THREAD

**Issue by BenzoIsHere** [bytecode77/pe-union#14]
Hello and thank you for reading and replying to my problem.
I've binded 2 files [one .exe around 40kb and one .txt around 3.5MB, both separated by an error appearing image]. I only selected encrypt, compress and hidden for the executable I'm interested in, and also selected ,,run as administrator'' execution in the ,exe's config prior to building it. After building it, zipping it with winrar and uploading it online, I tried to download it through google drive/anon files and many other upload sites, I can't even get to the point that I got it In my PC, can't say anything about even unzipping the folder. This is what i get:
![image](https://user-images.githubusercontent.com/90201064/132373063-e194eaf0-4f61-4c0c-94e5-1608f894f095.png)
Basically there's no chance that anyone could even try to open that folder. Can I have some advice? Bless.

**Comment by BenzoIsHere** [comment]
I get more detections on VirusTotal after encrypting it with pe-union than before even trying to encrypt it. What do you suggest

**Comment by bytecode77** [comment]
As already stated in the first paragraph: PEunion 4.0 is under active development while version 3.x is frozen. One of the reasons that the current version is detected is that it lacks in-memory execution support, which is very important in a crypter. The other reason is simply that this project is public & open source since 4 years.

PEunion 4.0 will support in-memory execution options and will have a completely new stub written in both C# and FASM including proper emulator detection and so on... Therefore, I ask everybody to await the new release. I will close this issue, because it is a duplicate of #8

All currently open issues are already addressed in version 4.0, which will be released soon.
